### DISCUSSION TITLE
마이크로 쉴드 에러 발생 문의

### DISCUSSION THREAD

**Discussion by swbaek81** [List-KR/community#73] [Ad-Shield]
안녕하세요 마이크로 쉴드가 아직도 에러가 종종 발생해서, 

그냥 Off 한채로 사용 한지 가 한 두 달은 된거 같습니다.

아이폰 인데요 4.12.0 버전에서도 종종 에러가 나오던지, 광고 차단이 안되던지 합니다.

너무 불편해서 적어 봅니다

**Comment by piquark6046** [comment]
@swbaek81 해당 되는 웹 사이트들을 모두 적어주세요.

**Comment by swbaek81** [comment]
개드립은 수시로 에러네요

https://www.dogdrip.net/586129034

![image](https://github.com/user-attachments/assets/23afcbb4-a2aa-40a2-bede-cc2a2d77e892)

**Comment by swbaek81** [comment]
와이고수 광고차단 자주 안 되는 케이스

https://m.ygosu.com/board/real_article/yeobgi/2093877/?searcht=&search=&page=1&type=normal

![image](https://github.com/user-attachments/assets/2123ce3b-1c71-4df6-a6c3-3a25300d8b26)

**Comment by swbaek81** [comment]
개드립 광고차단 안됨 잦음

https://www.dogdrip.net/585954492

![image](https://github.com/user-attachments/assets/f6b8c221-90c7-41ca-806d-abcea8f53444)

**Comment by swbaek81** [comment]
인벤 에러

https://m.inven.co.kr/board/webzine/2097/2329030?iskin=webzine

![image](https://github.com/user-attachments/assets/0dcb1f89-3e1f-4d93-834b-f4bc1961dd9f)

**Comment by swbaek81** [comment]
이슈야 에러

https://www.issuya.com/bbs/board.php?bo_table=issue&wr_id=1151242

![image](https://github.com/user-attachments/assets/561717bb-701a-4364-8990-97feef4768b0)

**Comment by swbaek81** [comment]
불안정해서 환불했던 유니콘프로를 다시 구입해볼까 심각하게 고려중입니다. 에러가 너무 많이 나서 마이크로쉴드는 못 써요

**Comment by swbaek81** [comment]
스포탈 코리아

https://m.sportalkorea.com/news/view.php?gisa_uniq=2024093017343394&sp=A&fy=&spnk=2024100717230926

![image](https://github.com/user-attachments/assets/e151cd03-2505-4098-abff-1ce0f4e9f2bd)
